### ISSUE TITLE
Streamfield issues IE11

### ISSUE THREAD

**Issue by lennarddevries** [wagtail/wagtail#5914]
### Issue Summary

While editing a page in IE11 I'm unable to add blocks to a StreamFields. Clicking the '+' button only triggers focus outline for the block above
### Steps to Reproduce

1. Set up fresh bakerydemo instance
2. Create a new blogpost instance
3. Add a block through the + button

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/7124952/77750214-b4572e00-7023-11ea-963d-8e006e1606d2.gif)

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: **yes**
### Technical details

* Python version: 3.6.0 / 3.6.9 .
* Django version: 2.2.9 / 3.0.4.
* Wagtail version: 2.7.1 / 2.8`.
* Browser version: IE11 (version: 11.379.17763.0)

**Comment by Stormheg** [comment]
I can reproduce this issue.

Here are my findings:

Using the select element tool I cannot hover the add button as the containing element behind it receives focus. This probably explains why clicking the button triggers the focus outline for the block.

<img width="1507" alt="Screenshot 2020-04-14 at 10 49 43" src="https://user-images.githubusercontent.com/13856515/79205136-d6cbb280-7e3d-11ea-87a5-a651e00737dc.png">

When I change the `pointer-events` property on the `.c-sf-add-button--visible` CSS class from `unset` to `visible` the button works again and I can add streamfields. It appears IE11 doesn't recognise `unset` as a valid property value.

<img width="1395" alt="Screenshot 2020-04-14 at 11 00 51" src="https://user-images.githubusercontent.com/13856515/79207281-cec14200-7e40-11ea-8bfd-f2f1420d058c.png">

I am short on time to investigate this issue in more depth but hopefully this is useful.

**Comment by paul-apacio** [comment]
I can second this as an issue.

**Comment by thibaudcolas** [comment]
I stumbled upon this as well and was about to create a new issue before I found this one. I wouldn’t be surprised if this was broken since the new StreamField UI was introduced in Wagtail 2.7, https://docs.wagtail.io/en/stable/releases/2.7.html#improved-streamfield-design.

It does look like the issue is with the elements’ stacking order as mentioned above, so might need to be resolved in the stylesheet for https://github.com/wagtail/react-streamfield, which is where those new styles are defined.

Tested in Wagtail 2.10 (current master) for me.

![ie11-streamfield-add](https://user-images.githubusercontent.com/877585/83762906-fb8b1c00-a66f-11ea-890c-61e9933e23b9.gif)

**Comment by thibaudcolas** [comment]
As a workaround it seems that interacting with that button with the keyboard works as expected, although unfortunately the button’s focus indicator isn’t visible so that’s not really something to rely upon.

**Comment by gasman** [comment]
This may well have been fixed in the telepath StreamField rewrite for 2.13 - while the CSS is largely unchanged, the HTML structure has been altered so that the '+' buttons are at the same level as the blocks, rather than each one being part of the block container above it. Would be good to have some feedback about whether this issue is still present...

**Comment by gasman** [comment]
Unfortunately it appears that the StreamField rewrite in 2.13 does _not_ fix this.

**Comment by lb-** [comment]
Due to #6170 likely happening in 2.15 I have removed the good first issue label

**Comment by gasman** [comment]
Closing as we will no longer be supporting IE11 as of 2.15.
